omnimatter_compression/prototypes/compress-buildings.lua

```diff
@@ -241,13 +241,131 @@ local process_fluid_box = function(fluid_box, i)
     end
   end
 end
+-- These names are ass
+local modspec = {
+  slot_count = "module_slots",
+  columns = "module_info_max_icons_per_row",
+  rows = "module_info_max_icon_rows",
+  shift = "module_info_icon_shift",
+  scale = "module_info_icon_scale",
+  gap_size = "module_info_separation_multiplier",
+  y_offset = "module_info_multi_row_initial_height_modifier"
+}
+-- Help us keep the code clean
+setmetatable(modspec, {
```

Comment:
Creating this access method was basically the only way I could keep things legible, and the performance difference is negligible